### PR TITLE
Added 2.2 Wrench to mods/Danowar

### PR DESCRIPTION
**Pull request by danowar**

### CODE REVIEW

File: mods/README.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ The list should be structured in the following way:
 
 - Mounts the Triangle Labs 2-to-1 adapter directly to the mosquito toolhead.
 
-[Tablet mount](../mods/Danowar/Tablet_Mount)
+[Tablet mount](../Mods/Danowar/Tablet_Mount)
```
Comment (TorinoFermic) [inline mods/README.md]: The folder path will not be correct and breaks this link in README.md

Comment (john--) [reply mods/README.md]: Yep, "mods" is now lower-case.

**Comment by FHeilmann** [comment]
@danowar any updates on this?

**Comment by danowar** [comment]
Should be updated now.  Sorry for the delay

**Comment by FHeilmann** [comment]
Hey, sorry about the incovenience, but we have moved all the mods to a separate repository.
Could you please resubmit your PR over there? (https://github.com/VoronDesign/VoronUsers) If not let me know and I'll add the files for you.

Sorry again!

-Florian

**Comment by danowar** [comment]
Florian,

Done, not a problem at all.  I had a conversation with doowrednu last week, and I suggested a separate repository for community mods, in effort to keep the "official" repository cleaner.

**Pull request closed without merging**